src/Shouldly.DiffEngine/DiffEngine.cs

```diff
@@ -0,0 +1,15 @@
+using DiffEngine;
+
+namespace Shouldly;
+
+public class DiffEngine : IDiffViewer
```

Comment:
Should this be in the Shouldly.DiffEngine namespace and be called DiffViewer ?

Reply:
I would say this is the `DiffEngineDiffViewer`, but tbh don't feel that strongly,

Reply:
Done